tests/by-util/test_ls.rs

```diff
@@ -3678,3 +3678,17 @@ fn test_ls_dired_complex() {
     println!("Extracted filenames: {:?}", filenames);
     assert_eq!(filenames, vec!["a1", "a22", "a333", "a4444", "d"]);
 }
+
+#[ignore = "issue #5396"]
+#[test]
+fn test_ls_tabsize_cf() {
```

Comment:
Not sure about the "tabsize" in the function name. The function is about testing whether there is a tab as a delimiter.
```suggestion
fn test_ls_cf_output_should_be_delimited_by_tab() {
```